scripts/drone/steps/lib.star

```diff
@@ -814,7 +814,10 @@ def build_docker_images_step(edition, ver_mode, archs=None, ubuntu=False, publis
     return {
         'name': 'build-docker-images' + ubuntu_sfx,
         'image': 'google/cloud-sdk',
-        'depends_on': ['copy-packages-for-docker'],
+        'depends_on': [
+          'copy-packages-for-docker',
+          'compile-build-cmd',
```

Comment:
This can be removed since `copy-packages-for-docker` already depends on `package` which depends on something that comes after the `compile-build-cmd` so it can be removed, but no blocker. We can remove it later 👍